sdk/core/Azure.Core.TestFramework/src/RunFrequencyAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+
+namespace Azure.Core.TestFramework
+{
+    [AttributeUsage(AttributeTargets.Assembly | AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
```

Comment:
Why `AllowMultiple=true`? That would imply the condition in your archetype-sdk-tests.yml would not work if a test could have multiple values.

Reply:
good catch, will update it